ArduCopter/RC_Channel.cpp

```diff
@@ -685,6 +691,29 @@ void RC_Channel_Copter::do_aux_function_change_force_flying(const AuxSwitchPos c
     }
 }
 
+#if QUICKTUNE_ENABLED
+// called on any Quicktune Aux function change
+void RC_Channel_Copter::do_aux_function_quicktune(const AuxSwitchPos ch_flag)
+{
+    if (copter.quicktune == nullptr && !copter.arming.is_armed()) {
```

Comment:
Use an early-return pattern here (and in the Plane equivalent) to simplify, e.g.:

```
    if (copter.quicktune == nullptr && copter.arming.is_armed()){
        GCS_SEND_TEXT(MAV_SEVERITY_WARNING, "Quicktune: Must be disarmed to initialise.");
       return;
    }
```

Reply:
removed